pkg/cvo/metrics.go

```diff
@@ -34,6 +35,7 @@ type operatorMetrics struct {
 	clusterOperatorConditions           *prometheus.GaugeVec
 	clusterOperatorConditionTransitions *prometheus.GaugeVec
 	clusterInstaller                    *prometheus.GaugeVec
+	clusterVersionOperatorRetrieving    *prometheus.GaugeVec
```

Comment:
nit: this internal property does not need the `clusterVersionOperator` namespacing that the metric name needs.  Can we just use `cannotRetrieveUpdates` or some such?

Reply:
I didn't use clusterVersionOperator due to namespace considerations but rather thought it best described what was being checked. Per status.md "...CVO is succesfully retrieving updates". I could make it CVORetrieving but think spelling it out better matches the pattern already being used. 
Do not want to use the negative named condition because it does not flow with the logic check of Status true.

Reply:
I think these are CVO/release metrics by default, unless they say otherwise.  So `version` would be CVO/release version.  `availableUpdates` is CVO/release updates.  This one would be `canRetrieveAvailableUpdates`.  Or, hmm, maybe it should be `availableUpdatesStaleSeconds` and record seconds since the last sync?